packages/jest-matchers/src/index.js

```diff
@@ -130,6 +130,12 @@ expect.extend = (matchersObj: MatchersObject): void => {
   Object.assign(global[GLOBAL_STATE].matchers, matchersObj);
 };
 
+expect.anything = global.jasmine.anything;
+expect.any = global.jasmine.any;
+expect.objectContaining = expect.objPart = global.jasmine.objectContaining;
+expect.arrayContaining = expect.arrPart = global.jasmine.arrayContaining;
+expect.stringMatching = expect.strMatch = global.jasmine.stringMatching;
```

Comment:
toBeCalled used to be a Jest specific matcher for the longest time until we rewrote the assertion library in Jest last summer and merged the two matchers. We couldn't just remove them as we didn't have enough time to run the codemod at FB. Please don't use one bad API as an excuse to add another alias.